README.md

```diff
@@ -25,5 +25,5 @@ Make sure to exclude `/tests` directories, to keep reporting classes that are us
 Many types are excluded by default, as they're collected by framework magic, e.g. console command classes. To exlude another class, e.g. your interface collector, use `--skip-type`:
 
 ```bash
-vendor/bin/class-leak check bin src --skip-type="App\\Contract\\SomeInterface"
+vendor/bin/class-leak check bin/ src/ --skip-type='App\Contract\SomeInterface'
```

Comment:
In shell backslashes do not work in single-quotes.

Reply:
This depends on OS and particular shell interpretter. The `--skip-type='App\Contract\SomeInterface'` is also good :+1: